### DISCUSSION TITLE
Search for gene / mRNA names?

### DISCUSSION THREAD

**Discussion by MikeAxtell** [GMOD/jbrowse-components#1484] [Q&A] [answered]
Given a track of type FeatureTrack , (a tabix-ed gff file), is there a way to index all of the gene and/or mRNA names to makes them searchable from the browser? I remember in JBrowse 1 this was a script that was run after loading the relevant track(s). I'm not seeing an equivalent in jbrowse2.

Thanks !,
Mike

**Comment by cmdcolin** [comment]
Name searching has not been added to JBrowse 2 yet unfortunately. We will probably end up porting some jbrowse 1 style search system sometime here...

**Reply by cmdcolin** [reply]
Just for a little more detail we are still deciding on just porting the jbrowse 1 perl code over to nodejs or to make a newish system, and a couple other things. If there are any features that would be wishlist items do let us know

**Comment by MikeAxtell** [comment, marked as answer]
Thanks you bet! 
Definitely the name/ID search I think would be very useful, however it's implemented.

**Comment by rbuels** [comment]
relevant issues in development:

#1163 
#56
